<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SportClass</class>
 <widget class="QMainWindow" name="SportClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>教练眼</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>340</y>
          <width>231</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>采集界面</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>320</y>
          <width>291</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>叠加对比界面</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>340</y>
          <width>281</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>动作全景界面</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>350</y>
          <width>281</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>二维解析界面</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>400</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分析界面</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>127</y>
      <width>117</width>
      <height>158</height>
     </rect>
    </property>
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>127</y>
      <width>153</width>
      <height>177</height>
     </rect>
    </property>
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>127</y>
      <width>117</width>
      <height>44</height>
     </rect>
    </property>
    <property name="title">
     <string>选择(S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>窗口(W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>127</y>
      <width>117</width>
      <height>44</height>
     </rect>
    </property>
    <property name="title">
     <string>采集(C)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>117</width>
      <height>44</height>
     </rect>
    </property>
    <property name="title">
     <string>分析(A)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>127</y>
      <width>117</width>
      <height>44</height>
     </rect>
    </property>
    <property name="title">
     <string>帮助(H)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_A"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="analy"/>
   <addaction name="contrast"/>
   <addaction name="all"/>
   <addaction name="dimension"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>采集文件</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>文件夹</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/1_caiji.png</normaloff>../../菜单栏/icon菜单栏/1_caiji.png</iconset>
   </property>
   <property name="text">
    <string>采集</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/2_bofang.png</normaloff>../../菜单栏/icon菜单栏/2_bofang.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="analy">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/3_fenxi.png</normaloff>../../菜单栏/icon菜单栏/3_fenxi.png</iconset>
   </property>
   <property name="text">
    <string>分析</string>
   </property>
  </action>
  <action name="contrast">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/4_diejiaduibi.png</normaloff>../../菜单栏/icon菜单栏/4_diejiaduibi.png</iconset>
   </property>
   <property name="text">
    <string>叠加对比</string>
   </property>
  </action>
  <action name="all">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/5_dongzuoquanjing.png</normaloff>../../菜单栏/icon菜单栏/5_dongzuoquanjing.png</iconset>
   </property>
   <property name="text">
    <string>动作全景</string>
   </property>
  </action>
  <action name="dimension">
   <property name="icon">
    <iconset>
     <normaloff>../../菜单栏/icon菜单栏/6_erweijiexi.png</normaloff>../../菜单栏/icon菜单栏/6_erweijiexi.png</iconset>
   </property>
   <property name="text">
    <string>二维解析</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>快照</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>设置图标全部选中</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Sport.qrc"/>
 </resources>
 <connections/>
</ui>
